### ISSUE TITLE
Are there any updates for soft-navs?

### ISSUE THREAD

**Issue by Coiven** [GoogleChrome/web-vitals#484]
I found that the last commit on this branch was two months ago, still based on `v3`, Will it be updated to `v4` in the future?

By the way, Origin Trails for soft-navigations will stop on 29th May. But our SPA website really needs to use soft-navigations to monitor performance data. So what can I do to continue using the ability to use soft-navigations after Origin Trials be ending(users don't need to manually open it through `chrome://flags`)?

**Comment by tunetheweb** [comment]
> I found that the last commit on this branch was two months ago, still based on `v3`, Will it be updated to `v4` in the future?

Yes, I'm aiming to do this over the next month. Quite a few changes, so might take me a bit of time.

> By the way, Origin Trails for soft-navigations will stop on 29th May. But our SPA website really needs to use soft-navigations to monitor performance data. So what can I do to continue using the ability to use soft-navigations after Origin Trials be ending(users don't need to manually open it through `chrome://flags`)?

As noted on the [Experimenting with measuring soft navigations post](https://developer.chrome.com/docs/web-platform/soft-navigations-experiment):

> *Update April 2024:** The [Soft Navigations origin trail](https://developer.chrome.com/origintrials#/view_trial/21392098230009857) will be ending on 29th May, after which the Chrome team will be taking time to review the experiment feedback and consider the next steps for this API. We want to address some accuracy issues and believe another origin trial will be required before the feature is ready to progress. In the meantime, developers can still experiment with the feature by enabling `chrome://flags/#enable-experimental-web-platform-features`.

Therefore it will not be possible to continue using soft navigations on your site until those next steps are announced.

**Comment by Coiven** [comment]
Thanks for your reply. so I should try another way to monitor my website until soft-navigations is released or start a new Origin Trial.

**Comment by tunetheweb** [comment]
Please see the latest updates here:
https://developer.chrome.com/blog/new-soft-navigations-origin-trial

Will move back to tracking this in #119
